docs/reference/indices/delete-data-stream.asciidoc

```diff
@@ -56,5 +56,5 @@ DELETE /_index_template/template
 
 `<data-stream>`::
 (Required, string)
-Name of the data stream to delete.
-Wildcard (`*`) expressions are supported.
+Comma-separated list of names or wildcard expressions of data streams to
+delete.
```

Comment:
I've been inconsistent with this, but `or` is technically incorrect as a list of wildcard expressions is supported. I'll open a PR to fix this in the other data stream API docs.
```suggestion
Comma-separated list of data streams to delete.
Wildcard (`*`) expressions are supported.
```